src/commands/flex/plugins/release.js

```diff
@@ -0,0 +1,59 @@
+const { flags } = require('@oclif/command');
+const { progress } = require('flex-plugins-utils-logger');
+
+const FlexPluginsCreateConfiguration = require('./create-configuration');
+const { createDescription } = require('../../../utils/general');
+
+/**
+ * Creates a Configuration
+ */
+class FlexPluginsRelease extends FlexPluginsCreateConfiguration {
+  constructor(argv, config, secureStorage) {
+    super(argv, config, secureStorage, { strict: false });
+
+    this.scriptArgs = [];
+  }
+
+  /**
+   * Main method
+   */
+  async doRun() {
+    if (this.flags.version && !this.flags.plugin) {
+      await this.doCreateRelease(this.flags.version);
+    } else {
+      const config = await super.doCreateConfiguration();
+      await this.doCreateRelease(config.version);
+    }
+  }
+
+  async doCreateRelease(version) {
+    await progress(`Enabling configuration **v${version}**`, async () => this.createRelease(version), false);
```

Comment:
would using the release wording be better?
```
Releasing configuration  **v${version}**
```
and
```
🚀 Your configuration **v${version}** was successfully released
```

Reply:
Asking Ruma to review

Reply:
We have been using the word enable in our docs to represent a configuration being released or being active on Flex. So I am okay with not using released.  Change the success message to "🚀 Configuration **v${version}** successfully enabled"